app/src/main/java/io/github/djunicode/canteenapp/LoginActivity.java

```diff
@@ -54,31 +54,31 @@ protected void onCreate(Bundle savedInstanceState) {
         token = retrieveToken();
         apiInterface = customRetrofit.create(ApiInterface.class);
 
-        email.addTextChangedListener(new MyTextWatcher(email));
-        password.addTextChangedListener(new MyTextWatcher(password));
-
-
-
-        loginButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-
-                if (!validateEmail()) {
-                    return;
-                }
-
-                if (!validatePassword()) {
-                    return;
-                }
-                if (email.getText().toString().equals("admin@email.com")  && password.getText().toString().equals("admin")){
-
-                    Toast.makeText(LoginActivity.this, "Logged in", Toast.LENGTH_SHORT).show();
-                    Intent i = new Intent(LoginActivity.this, MainActivity.class);
-                    i.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-                    startActivity(i);
-                }
-            }
-        });
+//        email.addTextChangedListener(new MyTextWatcher(email));
+//        password.addTextChangedListener(new MyTextWatcher(password));
+
+
+
+//        loginButton.setOnClickListener(new View.OnClickListener() {
+//            @Override
+//            public void onClick(View v) {
+//
+//                if (!validateEmail()) {
+//                    return;
+//                }
+//
+//                if (!validatePassword()) {
+//                    return;
+//                }
+//                if (email.getText().toString().equals("admin@email.com")  && password.getText().toString().equals("admin")){
+//
```

Comment:
Do we need this commented code?

Reply:
Ya, this was written for the username being the email, now its a just the sap id

Reply:
So, if finally sap is being used instead of email shouldn't we change the variable names accordingly?

Reply:
I'll make the necessary changes along with the next pr